<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveBox</class>
 <widget class="QWidget" name="CurveBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>63</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QwtPlot" name="qwtPlot"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_options">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QToolButton" name="btn_Load">
          <property name="text">
           <string>open</string>
          </property>
          <property name="icon">
           <iconset resource="../../icons/grapher.qrc">
            <normaloff>:/grapher/fileopen.png</normaloff>:/grapher/fileopen.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_Save">
          <property name="text">
           <string>save</string>
          </property>
          <property name="icon">
           <iconset resource="../../icons/grapher.qrc">
            <normaloff>:/grapher/filesave.png</normaloff>:/grapher/filesave.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/grapher.qrc"/>
 </resources>
 <connections/>
</ui>
